examples/boltzmann_wealth_model/app.py

```diff
@@ -0,0 +1,127 @@
+import mesa
+
+from boltzmann_wealth_model.model import BoltzmannWealthModel
+
+import streamlit as st
+
+import time
+
+import pandas as pd
+
+import altair as alt
```

Comment:
What is the reason for switching to `altair`?

Reply:
I could not do the animation of the grid in plotly.